test/e2e/proxy_test.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"encoding/hex"
+	"fmt"
+	"net"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func proxyEnabled(data *TestData) (bool, error) {
+	key := "resubmit(,40),resubmit(,41)"
```

Comment:
as a follow-up to an earlier comment I made, there is already a `GetAntreaConfigMap` method defined for e2e tests, but I am not sure it's a much better solution than what you are doing at the moment

Reply:
If we use the configmap method, then we will need to parse the config.
For now, I prefer to keep the current code.
In the future, as we may add more experimental features, we can have a particular function to retrieve featuregayte options in e2e tests. How do you think about this?

Reply:
Sounds good